### DISCUSSION TITLE
How can I get the old models back?

### DISCUSSION THREAD

**Discussion by AshleighTheCutie** [fnuecke/oc2#287] [General]
I'd love to use it, but it looks off and... just kinda bad. Has anyone ported the original block models over?

**Comment by atirutw** [comment]
Hot take, I think the new models are pretty good

**Comment by CoolCat467** [comment]
I would guess you could likely make a resource pack? I don't think you could easily replace the models without another mod to do that specifically, but you can replace the textures with different ones through minecraft's texture pack system. I don't remember seeing anything here about that, but I could be wrong. Best of luck to you!

**Reply by samuelh2005** [reply]
You can replace the models and textures with a resource pack. Just make sure to provide JSON files with the same path as the current ones.
